storage/blob.go

```diff
@@ -633,7 +237,7 @@ func (b BlobStorageClient) BlobExists(container, name string) (bool, error) {
 // specified container. This method does not create a publicly accessible URL if
 // the blob or container is private and this method does not check if the blob
 // exists.
-func (b BlobStorageClient) GetBlobURL(container, name string) string {
+func (b BlobServiceClient) GetBlobURL(container, name string) string {
```

Comment:
This change got revoked before #532 got merged. Those changes should be refactored back here as well.